x-pack/plugins/alerting/server/rules_client/tests/bulk_edit.test.ts

```diff
@@ -302,6 +306,7 @@ describe('bulkEdit()', () => {
             type: 'alert',
             attributes: expect.objectContaining({
               tags: [],
+              revision: 1,
```

Comment:
This is a part of the test `should delete tag`.

I don't get why the revision gets incremented here from 0 to 1, while the test doesn't seem to be deleting a tag that actually exists in a rule.